### ISSUE TITLE
Feedback Requested - Grey/whitelists and the app in general

### ISSUE THREAD

**Issue by MrEliasen** [MrEliasen/twitch-bot-list#40]
TL;DR: If you don't want to comment I can provide a few options to choose from, comment on this with an emote:
- 🌵 : Keep it as it
- 💧 : Remove grey/whitelist and only allow users to upload lists manually
- 🎈 : Keep grey/whitelist and only allow users to upload lists manually

(I will add more if suggestions comes in)

Hi all,

This repository and the accompanied web app have been getting a lot of attention (both negative and positive) lately, so I thought I would try address some of the changes by also ask for feedback on how to proceed.

This repository It has outgrown its original intent and purpose (since release 2 years ago) so it is time to make some changes.

**Actioned Changes**

1. I have completely removed the old bot list since it was just not maintainable and also I don't want to be the arbiter of such list.  The web app will now directly fetch from Twitch Insights, which I have no involvement in. I am still maintaining a grey and whitelist for the time being, which you can still requests to add/remove accounts from.
2. The tool allows you to upload a list manually, completely ignoring Twitch Insights.
3. I have added a lot more warnings and general text to the site in hope it might add a bit more context and information about the tool and how it works.

**Possible Future Changes**

At minimum I will keep the tool online and just allow users to upload their own  lists manually, but without a grey or whitelist. This is also my personal ideal solution as it completely removes this repository/myself from whatever the latest issues are with Twitch users/bots/etc.

However, I would still like to give the opportunity to provide feedback/suggestions.

If you do decide to comment:
- Please refrain from targeting any individuals
- Keep the language civil
- Do not state opinions as fact
- Do not include political topics or similar.

Thank you!

**Comment by StygianStyx** [comment]
Can my name please just be removed or added to some other list?

**Comment by TGxNightshade** [comment]
In addition to my vote, I say @MrEliasen needs to be replaced by someone who has an actual interest in this, both now and in the future. Since you've made it so apparent you don't care, find someone who does.

**Comment by MrEliasen** [comment]
@TGxNightshade Go cancel somewhere else.

**Comment by BLKHeroesEverywhere** [comment]
I was voted from the admin team to do this.

Styx is actually a really cool dude, he has helped our Facebook group immensely by expanding our socials. Hell we even made him an honorary mod (The only white mod/admin in a group of over 8k Black users).

I would support removing his name. He really is just using it as a networking tool https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-926808400 and helps people he meets, He has all his socials linked on his account also so its not like it would be hard to get ahold of him.

**Comment by MrEliasen** [comment]
> I was voted from the admin team to do this.
> 
> Styx is actually a really cool dude, he has helped our Facebook group immensely by expanding our socials. Hell we even made him an honorary mod (The only white mod/admin in a group of over 8k Black users).
> 
> I would support removing his name. He really is just using it as a networking tool [#39 (comment)](https://github.com/MrEliasen/twitch-bot-list/issues/39#issuecomment-926808400) and helps people he meets, He has all his socials linked on his account also so its not like it would be hard to get ahold of him.

Hey, I don't want to sound disrespectful, but I keep saying.. there is no list to remove anyone from any more..
I don't understand how hard that is to understand. There has been an entire thread about this which devolved into smears because people don't read or don't want to understand what actually is going on.

There is no list, there used to be, but there is no list anymore.

I hope this clears it up once and for all. In case you need to see to believe:
https://github.com/MrEliasen/twitch-bot-list/blob/master/list.json

----

Please keep this thread on the topic. Any comments in this thread which are not related to the topic will be removed from this point out.

**Comment by BLKHeroesEverywhere** [comment]
What about this? https://github.com/MrEliasen/twitch-bot-list/issues/40#issuecomment-927326919 Atleast Give My Boy and the rest like him to have a option to be moved if asked.

**Comment by MrEliasen** [comment]
For the sake of clearing this up, I will allow this to continue between you and me only for a moment @BLKHeroesEverywhere

> What about this? [#40 (comment)](https://github.com/MrEliasen/twitch-bot-list/issues/40#issuecomment-927326919) Atleast Give My Boy and the rest like him to have a option to be moved if asked.

That comments just said to add him to some 4th list. There are 3 lists:
- "the list" which is now empty and no longer in use.
- Greylist which is for for those who wish to be added to it, but it might draw more bad attention than not, and is the optional ban list basically.
- Whitelist: Reserved purely for verified opt-in only bots.

Anyone are welcome to create a PR to add/remove accounts of these lists if there they can argue why.

That is all it is...

**Comment by BLKHeroesEverywhere** [comment]
This tool lets you 1 click ban him.

https://ban-twitch-bots.sirmre.com/

**Comment by MrEliasen** [comment]
https://i.imgur.com/MyooTF2.png

**Comment by MrEliasen** [comment]
Make 4th list to do what exactly? You are asking me to just make a list and put names in it, not what it should do, how it should work.

This is what I am saying, I do not control the list. See above image.

I can add to a gray list, just make a PR. I don't know how many time I can repeat these 2 things.

**Comment by BLKHeroesEverywhere** [comment]
A 4th List of Opt-In Real users doing whatever he is doing to network, one that does not list them as Bots and have their names auto checked to just ban.

Our account is in that list to by the way.

**Comment by MrEliasen** [comment]
You literally are describing the greylist, and you clearly did not read or understand what I highlighted in the image.
This is going in circles. If he wants to be put on a list, that is the graylist. Create a PR, easy.

**Comment by BLKHeroesEverywhere** [comment]
I don't know what a PR is but he has clearly asked to be added to any alternative. As do we.

We are a black community for gamers, comic lovers, and nerds this man helped us with our community and we wanted to use what he did for us to help other black content creators, instead we have been listed as a bot to just auto ban. Im furious and im trying my best to be respectful. Please look at this from a Community perspective. Its obvious all he is doing is helping people out.

**Comment by StygianStyx** [comment]
yes plz, i just dont want my name to be a 1 click ban already selected. again i have to speak up for these dudes also https://github.com/MrEliasen/twitch-bot-list/issues/40#issuecomment-927326919

**Comment by StygianStyx** [comment]
The Fact that i have to keep begging every day to be removed is sad dude, please remove my name or add it to some other list. Im Going to be DMing this and the other list to twitch support. not sure what they can do but this has to be some form of harassment.

**Comment by MrEliasen** [comment]
- https://github.com/MrEliasen/twitch-bot-list/issues/40#issuecomment-930420439
- https://github.com/MrEliasen/twitch-bot-list/issues/40#issuecomment-930409313

The only problem here is you either dp not read or understand what I keep telling you.
I will no longer respond to that subject.

**Comment by StygianStyx** [comment]
you said all i hyave to do is ask to be move, i want my name in the gay list or white list, im tired of being auto checked to be banned

**Comment by MrEliasen** [comment]
Holy crap we have a breakthrough!

No problem, create a pull request with your changes. If you do not know how to do that please create a new issue and fill out the form, stating you want to have your name added to the graylist.

**Comment by StygianStyx** [comment]
Dude why is it so hard. i been asking. it was the 1st question i asked. no i do not know how.. but i have asked several times.

**Comment by MrEliasen** [comment]
> Dude why is it so hard. i been asking. it was the 1st question i asked. no i do not know how.. but i have asked several times.

You have asked to be removed from a list since the start. The list got nuked as it was obsolete and unmaintained. You then asked to be added to any list or for me to create some random other list to add people to without stating wtf how that list would work or what the purpose would be which the Greylist does not already do.

When I did ask you if you wanted to be added to the Graylist you either didnt reply or asked to get removed from the bot list, which I do not control.

It wouldn't have been this hard if you would read what I actually wrote to you, over and over, and yet here we are and you STILL don't

> If you do not know how to do that please create a new issue and fill out the form, stating you want to have your name added to the graylist.

That was literally the last half of the sentence I replied to you with just now, and yet you decide to instead reply here and rant.

So. to confirm, you want to be added to the greylist? Because You also didn't confirm that in your last message.

A simple yes/no will do at this point.

**Comment by StygianStyx** [comment]
Yes i would like to be added to the Greylist.

**Comment by BLKHeroesEverywhere** [comment]
Please add us to the grey list as well.

**Comment by MrEliasen** [comment]
> Please add us to the grey list as well.

What is your twitch name please? To make sure I get it right.

**Comment by BLKHeroesEverywhere** [comment]
BLKHeroesEverywhere

**Comment by djwikkid** [comment]
I am having issue with banning the bots. I'm logged in and see my profile Twitch pic in the upper right corner. I select all I want to ban, click the red ban button and nothing.  the banned counter still reads 0/106. Refresh and get the same.
